google/cloud/bigtable/CMakeLists.txt

```diff
@@ -12,6 +12,91 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This should be included from the top level CMakeLists file
+set(BIGTABLE_CLIENT_VERSION_MAJOR 0)
+set(BIGTABLE_CLIENT_VERSION_MINOR 2)
+set(BIGTABLE_CLIENT_VERSION_PATCH 0)
+
+# Configure the Compiler options, we will be using C++11 features.
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+set(DOXYGEN_PROJECT_NAME "Google Cloud Bigtable C++ Client")
+set(DOXYGEN_PROJECT_BRIEF "A C++ Client Library for Google Cloud Bigtable")
+set(DOXYGEN_PROJECT_NUMBER
+        "${BIGTABLE_CLIENT_VERSION_MAJOR}.${BIGTABLE_CLIENT_VERSION_MINOR}.${BIGTABLE_CLIENT_VERSION_PATCH}")
+set(DOXYGEN_EXAMPLE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/examples)
+set(DOXYGEN_PREDEFINED "BIGTABLE_CLIENT_NS=v${BIGTABLE_CLIENT_VERSION_MAJOR}")
+set(DOXYGEN_EXCLUDE_PATTERNS
+        "*/google/cloud/bigtable/README.md"
+        "*/google/cloud/bigtable/benchmarks/*"
+        "*/google/cloud/bigtable/ci/*"
+        "*/google/cloud/bigtable/examples/*.md"
+        "*/google/cloud/bigtable/internal/*"
+        "*/google/cloud/bigtable/testing/*"
+        "*/google/cloud/bigtable/tests/*"
+        "*/google/cloud/bigtable/tools/*"
+        "*/google/cloud/bigtable/*_test.cc")
+
+include(GoogleCloudCppCommon)
```

Comment:
Can this import hold the aforementioned version number settings?

Reply:
Bigtable is at 0.2.0 (the master branch is) because we already "released" 0.1.0.  The rest of the libraries are at 0.1.0.  So no?

Reply:
SGTM